Minor updates to Access Review Beta (#14738)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/accessreviewschedulesettings.md

| applyActions|[accessReviewApplyAction](../resources/accessreviewapplyaction.md) collection | Optional field. Describes the  actions to take once a review is complete. There are two types that are currently supported: `removeAccessApplyAction` (default) and `disableAndDeleteUserApplyAction`. Field only needs to be specified in the case of `disableAndDeleteUserApplyAction`. |
| recommendationsEnabled|Boolean | Indicates whether decision recommendations are enabled or disabled. |
| recommendationLookBackDuration | Duration| Optional field. Indicates the time period of inactivity (with respect to the start date of the review instance) that recommendations will be configured from. The recommendation will be to `deny` if the user is inactive during the look back duration. For reviews of groups and Azure AD roles, any duration is accepted. For reviews of applications, 30 days is the maximum duration. If not specified, the duration is 30 days. |

> | recommendationInsightSettings|[accessReviewRecommendationInsightSetting](../resources/accessReviewRecommendationInsightSetting.md) collection | Optional field. Describes the types of insights that aid reviewers making decisions to include in the review. There is one type that is currently supported: `userLastSignInRecommendationInsightSetting` (default) |

Reviewer: Too "early" for the statement on the supported derived type. That'll be in the accessReviewRecommendationInsightSetting resource description.
```suggestion
| recommendationInsightSettings|[accessReviewRecommendationInsightSetting](../resources/accessReviewRecommendationInsightSetting.md) collection | Optional. Describes the types of insights that aid reviewers to make access review decisions. |
```

    }
  ],
  "recommendationsEnabled": "Boolean",
  "recommendationLookBackDuration": "Duration",
  "recommendationInsightSettings": [
    {
      "@odata.type": "microsoft.graph.accessReviewRecommendationInsightSetting"
    }

>   ],

Reviewer: ```suggestion
  ]
```
